### DISCUSSION TITLE
I couldn't find the file /etc/logstash/

### DISCUSSION THREAD

**Discussion by omdevops99** [learndevopsonline/learndevopsonline#1614] [Q&A] [answered]
Hi Team,

I couldn't find the file /etc/logstash/ after installing the ELK.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/de0ad029-f457-4994-9202-84d3df66c119)
Public IP: 3.239.127.107

**Comment by sreedharm07** [comment, marked as answer]
hi @omswetha 
is there give  **cat /etc/logstash/logstash.yml**. you given **cat /etc/logstash/**
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/d5e8ba2d-c7f5-44de-b2a9-94fb1027404e)

**Reply by omdevops99** [reply]
Thanks got it!
